### PR TITLE
HRM-86: Added address to applicant

### PR DESCRIPTION
**Pull request by spolischook**
https://magecore.atlassian.net/browse/HRM-86
- Added AddressFormType
- Added migration
- Updated view and update template

### CODE REVIEW

**Comment by spolischook** [comment]
@yurio please review this

File: src/Oro/Bundle/HRMBundle/Entity/Applicant.php [hunk 1]
```diff
@@ -189,9 +190,10 @@ public function __construct()
     protected $organization;
 
     /**
-     * @var Collection
+     * @ORM\OneToOne(targetEntity="Oro\Bundle\AddressBundle\Entity\Address", cascade={"persist"})
+     * @ORM\JoinColumn(name="address_id", referencedColumnName="id", onDelete="SET NULL")
```
Comment (yurio) [inline src/Oro/Bundle/HRMBundle/Entity/Applicant.php]: You set onDelete="SET NULL". As result, during deleting the address record will be useless as for me. I think it's better to set CASCADE in this case. What do you think?

File: src/Oro/Bundle/HRMBundle/Resources/views/Applicant/update.html.twig [hunk 2]
```diff
@@ -37,17 +37,25 @@
...
-                form_row(form.lastName),
-                form_row(form.birthday),
-                form_row(form.email),
-                form_row(form.technologies),
-                form_row(form.contactList),
-            ]
-        }]
+        'subblocks': [
+            {
+                'title': 'Personal Information',
```
Comment (yurio) [inline src/Oro/Bundle/HRMBundle/Resources/views/Applicant/update.html.twig]: This title is not translatable

File: src/Oro/Bundle/HRMBundle/Resources/views/Applicant/update.html.twig [hunk 3]
```diff
@@ -37,17 +37,25 @@
...
+                    form_row(form.firstName),
+                    form_row(form.lastName),
+                    form_row(form.birthday),
+                    form_row(form.email),
+                    form_row(form.technologies),
+                    form_row(form.contactList),
+                ]
+            },
+            {
+                'title': 'Primary Address',
```
Comment (yurio) [inline src/Oro/Bundle/HRMBundle/Resources/views/Applicant/update.html.twig]: The same as above

File: src/Oro/Bundle/HRMBundle/Entity/Applicant.php (hunk 1 above)
Comment (spolischook) [inline src/Oro/Bundle/HRMBundle/Entity/Applicant.php]: Yea!

**Pull request merged**